enterprise/dev/ci/internal/ci/bazel_operations.go

```diff
@@ -147,6 +147,14 @@ func bazelTest(targets ...string) func(*bk.Pipeline) {
 
 	// Test commands
 	bazelTestCmds := []bk.StepOpt{}
+
+	// bazel build //client/web:bundle is very resource hungry and often crashes when ran along other targets
+	// so we run it first to avoid failing builds midway.
+	cmds = append(cmds,
+		bazelAnnouncef("bazel build //client/web:bundle"),
+		bk.Cmd(bazelCmd("build //client/web:bundle")),
```

Comment:
Would it be safer to build both webpack targets first? (`:bundle` and `:bundle-enterprise`)

```suggestion
		bazelAnnouncef("bazel test //client/web:webpack_test"),
		bk.Cmd(bazelCmd("test //client/web:webpack_test")),
```

Reply:
It's the build part that crashes, I'd rather keep the tests with the others. We can revisit this if it's not stable enough, WDYT?

Reply:
@jhchabran, sounds good! Still, building `:bundle-enterprise` would be helpful here because it's the slowest and biggest webpack build we have.

Reply:
👍